assets/prometheus-k8s/prometheus-k8s.yaml

```diff
@@ -12,6 +12,9 @@ spec:
   serviceMonitorSelector:
     matchExpressions:
     - {key: k8s-app, operator: Exists}
```

Comment:
I would say we should just change this `serviceMonitorSelector` to the "all" selector (`{}`). `k8s-app` is an old artifact of ours that we should get rid of.

Reply:
Agree, also commented on this [elsewhere](https://github.com/openshift/openshift-ansible/pull/8518#discussion_r190882403). I vote followup for now.